pmm/pmm2-server-release.groovy

```diff
@@ -182,17 +182,17 @@ pipeline {
                         docker save percona/pmm-server:\${VERSION} | xz > pmm-server-\${VERSION}.docker
 
                         docker pull \${DOCKER_CLIENT_VERSION}
-                        docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${VERSION}
-                        docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${DOCKER_MID}
-                        docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${TOP_VER}
+                        docker tag \${DOCKER_CLIENT_VERSION} percona/pmm-client:\${VERSION}
+                        docker tag \${DOCKER_CLIENT_VERSION} percona/pmm-client:\${DOCKER_MID}
+                        docker tag \${DOCKER_CLIENT_VERSION} percona/pmm-client:\${TOP_VER}
                         if [ \${TOP_VER} = 1 ]; then
-                            docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:latest
-                            docker push perconalab/pmm-client:latest
+                            docker tag \${DOCKER_CLIENT_VERSION} percona/pmm-client:latest
+                            docker push percona/pmm-client:latest
```

Comment:
are we going to issue pmm-client for version 1 to percona account too?

Reply:
If I remember correctly, we don't have a docker for `percona/pmm-client`  at all . 
So `:latest` will be  -  adding the new one. 

pls, not introduce it right now. There will be a task in Jan. to switch `percona/pmm-server:latest` to  `pmm-server:2` . And at the same time, we'll introduce `:latest`  for pmm-client.


As for now, we need `:2`  ,  `:2.12` and `2.12.0` (as examples ) tags only. The same way we do them for pmm-server